stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -7,29 +7,34 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/bottom_sheet"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
What are the scenarios where we need to `match_parent` vs `wrap_content`?

Reply:
I think generally we want to `match_parent` whenever we're showing an input form - today just the CardMultilineWidget - so that we can expand to full screen.

For the saved PMs list (and the loading spinner), we want to wrap content as it doesn't really make sense to expand here.